examples/datadog/agent_on_gke_autopilot_values.yaml

```diff
@@ -0,0 +1,53 @@
+# Datadog Agent on GKE Autopilot cluster
+
+## On GKE Autopilot, registry needs to be GCR
+registry: gcr.io/datadoghq
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  clusterName: <CLUSTER_NAME>
+  tags: []
+  logs:
+    enabled: true
+    containerCollectAll: true
+    containerCollectUsingFiles: true
+  apm:
+    portEnabled: true
+  processAgent:
+    enabled: true
+  systemProbe:
+    enableTCPQueueLength: false
+    enableOOMKill: true
```

Comment:
`system-probe` isn't supported with GKE Autopilot, can set `datadog.systemProbe.enableOOMKill: false` to avoid deploying that.

Reply:
Removed it to not deploy it. Tested in my env.